bears/vcs/CommitBear.py

```diff
@@ -73,7 +75,7 @@ def setup_dependencies(self):
                 nltk.download([
                     'punkt',
                     'averaged_perceptron_tagger'
-                ])
+                ], print_error_to=sys.stdout)
```

Comment:
I expect this should be `sys.stderr`

Reply:
`redirect_stderr` is not available on python 3.4.

Reply:
ok, lets do this for now and see if it causes other problems.
Im especially worried about `coala --json`